src/builder.rs

```diff
@@ -36,6 +36,15 @@ impl BridgeBuilder {
         builder
     }
 
+    pub fn with_redirect_policy(self, policy: Policy) -> Self {
```

Comment:
is it ok to expose "Policy" directly ("reqwest::redirect::Policy") or we wanna remap it to our own type?

Reply:
I'm thinking if this make sense at all, I'll explain myself better, as we are using Reqwest exposing their type, is not just useless to rexpose all the reqwest-centric types and parameter?

In fact, we could think something like passing a custom `Reqwest` object entirely created by us, a method `with_client_builder` but I think is wrong as we constrain a relationship with the library and our application, forcing to use the same version of the thirdy-part library as well.

We should decouple this binding using a new custom type on this library, this will also cover us in case any update of the methods of Reqwest as we could just update the library when needed without breaking our code (hexagonal way!) 😄

Reply:
I second mapping it to an internal type to limit the kind of policies we actually want to support and decouple from Reqwest

Reply:
I agree with @cpiemontese too

Reply:
+1 on decoupling the with an our defined type